csrc/device_lower/analysis/predicate_elimination.cpp

```diff
@@ -948,13 +948,6 @@ class PredicateChecker : public IterVisitor {
     }
   }
 
-  // Disable predicate elimination for now. Should be unnecessary if
```

Comment:
This is no longer necessary as TopKOp always initializes its input and in that case it's taken care at https://github.com/NVIDIA/Fuser/blob/main/csrc/device_lower/analysis/predicate_elimination.cpp#L456-L466.